models/tridimensional/data_kleinstiver/Analysis of PyRosetta Output.py

```diff
@@ -0,0 +1,72 @@
+
+import scipy, scipy.stats.kendalltau, scipy.stats.chisquare
+
+def los_clnr(los):
```

Comment:
how about csv_row_parser or csv_transform or csv_parser, this fn name makes me think of LA lol